src/Core/pages/time_of_day.py

```diff
@@ -2,44 +2,73 @@
 from dash import Dash, html, dcc, callback, Output, Input
 from plotly.graph_objs import Figure
 import plotly.express as px
+import numpy as np
 
+from src.Core.data_provider import get_df, get_category_orders
+from src.Core.styles import graphDivStyle, pageStyle, graphStyle, legendColors
 
-from src.Core.data_provider import get_df
-from src.Core.styles import graphDivStyle, pageStyle
-
-dash.register_page(__name__)
+dash.register_page(__name__, name="When does injuries happen?")
 
 df = get_df()
 
+eventsListRaw = df["Event Type Group"].unique()
+eventsList = np.delete(eventsListRaw, np.where(eventsListRaw == "Non-RGX Collision"))
```

Comment:
I believe this is old, since we clean NON-RGX in data cleaner. If this is removed later on in the code where eventsList is used, the code should use df["Event Type Group].unique()